tests/unit_tests/embedding/test_rwkv.py

```diff
@@ -0,0 +1,12 @@
+from gptcache.embedding import Rwkv
+from gptcache.adapter.api import _get_model
+
+
+def test_rwkv():
```

Comment:
mark the test case to l2, add the annotation. `@pytest.mark.tags("L2")`

Reply:
okay